consensus/src/block_storage/block_store.rs

```diff
@@ -56,6 +56,11 @@ pub fn update_counters_for_committed_blocks(blocks_to_commit: &[Arc<ExecutedBloc
         counters::LAST_COMMITTED_ROUND.set(block.round() as i64);
         counters::LAST_COMMITTED_VERSION.set(block.compute_result().num_leaves() as i64);
 
+        let failed_rounds = block.block().block_data().failed_authors().map(|v| v.len()).unwrap_or(0);
```

Comment:
Can you clarify how is it different from `aptos_consensus_timeout_rounds_count`?

Reply:
that one skips rounds based on how we receive certificates. you can see the discussion above